src/main/java/net/sf/mcf2pdf/mcfelements/util/ImageUtil.java

```diff
@@ -83,7 +83,8 @@ private ImageUtil() {
 					return new float[] { DEFAULT_RESOLUTION, DEFAULT_RESOLUTION };
 				}
 				double[] dpi = meta.getDpi();
-				return new float[] { (float)dpi[0], (float)dpi[1] };
+				return new float[] { DEFAULT_RESOLUTION, DEFAULT_RESOLUTION };
+				//return new float[] { (float)dpi[0], (float)dpi[1] };

```

Comment:
Please do not comment out existing lines; we have the GitHub diff to see what has changed :-)
If you want to restore it one day, you can just go back in history. Please just remove commented out line. Thanks!